deploy/ansible/roles/base/tasks/main.yml

```diff
@@ -2,6 +2,9 @@
 # or the project is cloned or built. All system dependencies should be
 # installed here.
 
+- name: ensure all packages are upgraded safely
+  apt: upgrade=safe update_cache=yes
```

Comment:
The last run is cached for 5 minutes by default even if it is called across multiple tasks.